README.md

```diff
@@ -13,58 +13,130 @@ In your Meteor app directory, enter:
 $ meteor add aldeed:schema-index
 ```
 
-## Usage
+## Migration from 1.0.x to 2.0.x
+Version 1.0.x used index definitions exclusively on a collection's simpleschema.  Version 2 moves some of the index properties to an external function call (`attachIndex`).  Migration from 1.x to 2.x requires modificaiton of the simpleschema and addition of a call to `attachIndex` for each index to create.  For example in version 1.0.x:
 
-Use the `index` option to ensure a MongoDB index for a specific field:
+```js
+var books = new Mongo.Collection('books');
+books.attachSchema(new SimpleSchema({
+  title: {
+    type: String,
+    label: 'Title',
+    max: 200,
+    index: true
+  },
+  author: {
+    type: String,
+    label: 'Author'
+  },
+  isbn: {
+    type: String,
+    label: 'ISBN',
+    optional: true,
+    index: 1,
+    unique: true
+  },
+  //...
+}));
+```
+
+In version 2.0.x:
 
 ```js
-{
+var books = new Mongo.Collection('books');
+books.attachSchema(new SimpleSchema({
   title: {
     type: String,
-    index: 1
+    label: 'Title',
+    max: 200,
+    index: {
+      name: 'title',
+      type:-1
+    }
+  },
+  author: {cccccccblegetujrrtbgfrcncllvketghefdehvhiuln
```

Comment:
what's that? :)

Reply:
looks like a yubikey OTP 

cccccchcdhvtiklejrnuujjbtedbbnllktvgigvtnfrr

:)